compiler/qsc_rca/tests/callables.rs

```diff
@@ -142,7 +142,7 @@ fn check_rca_for_operation_with_one_classical_return_and_one_dynamic_return() {
             Callable: CallableComputeProperties:
                 body: ApplicationsGeneratorSet:
                     inherent: Quantum: QuantumProperties:
-                        runtime_features: RuntimeFeatureFlags(UseOfDynamicBool | ForwardBranchingOnDynamicValue | ReturnWithinDynamicScope)
+                        runtime_features: RuntimeFeatureFlags(UseOfDynamicBool | ReturnWithinDynamicScope)
```

Comment:
Why did forward branching capability get removed here? Shouldn't it still require that since there is a branch on measurement result?

Reply:
Oh, whoops... that's the intent of the change! I should read the other files too ;)